.github/workflows/_copyright_check.yml

```diff
@@ -15,6 +15,13 @@ name: Copyright check
 
 on: 
   workflow_call:
+    inputs:
+      other-file-types-to-search:
+        required: false
+        description: "Command for other File types to search copyright"
+        type: string
+        #sample input string value: "-or -name *.yaml"
+        #value: "-or -name *.yaml"
```

Comment:
The name `other-file-types-to-search` is misleading: Either this needs to be named `additional-find-args`, or better:

Let it be a comma-separated list of file types (`yaml,py,txt`), which you iterate over for creating the file-arguments dynamically